report/delayedusers/classes/report.php

```diff
@@ -131,7 +131,7 @@ public function get_submissions_sql() {
         $submissiontable = 'SELECT userid, surveyproid
```

Comment:
I want to write a query to select the list of user enrolled in the course but still not submitting a surveypro.
The query that caught your attention is part of a bigger query that looks like:

SELECT u.id, u.email, 
       s.surveyproid 
FROM m32_user u 
    LEFT JOIN (SELECT userid, surveyproid 
               FROM m32_surveypro_submission
               WHERE surveyproid = 156
               GROUP BY userid, surveyproid) s ON s.userid = u.id 
    JOIN (SELECT DISTINCT eu1_u.id 
          FROM m32_user eu1_u 
              JOIN m32_user_enrolments ej1_ue ON ej1_ue.userid = eu1_u.id 
              JOIN m32_enrol ej1_e ON (ej1_e.id = ej1_ue.enrolid AND ej1_e.courseid = :ej1_courseid) 
          WHERE 1 = 1 
              AND eu1_u.deleted = 0) eu ON eu.id = u.id 
WHERE surveyproid IS NULL 
ORDER BY lastname DESC